app/enterprise/0.33-x/admin-gui/configuration/managing-admins.md

```diff
@@ -61,10 +61,10 @@ HTTP/1.1 200 OK
 
 Now that you have the `<CONSUMER_ID>`, you can use this to create a credential with your associated `admin_gui_auth` plugin:
 
-* [Create a Basic Authentication Credential Username/Password](/enterprise/{{page.kong_versions}}/admin-gui/configuration/authentication/#enable-authentication)
-* [Create a Key Authentication Credential Key](/enterprise/{{page.kong_versions}}/admin-gui/configuration/authentication/#basic-authentication)
+* [Create a Basic Authentication Credential Username/Password](/enterprise/{{page.kong_version}}/admin-gui/configuration/authentication/#enable-authentication)
```

Comment:
This is the wrong anchor tag - it is pointing to #enable-authentication and should probably be pointing to #basic-authentication